inference.sh

```diff
@@ -0,0 +1,15 @@
+rm -rf alignments/
```

Comment:
@Fazziekey , Hi, I think it's better not to commit the shell script to the repo.